insurance/modelmonitor.ipynb

```diff
@@ -37,7 +39,48 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "api.import_deployment(name=MONITOR_NAME)"
+    "def wait_for_deployment_running(deployment_id):\n",
+    "    status = None\n",
+    "    inference_url, inference = None, None\n",
+    "    while True:\n",
+    "        data = api.get_deployment(deployment_id)\n",
+    "        status = data.data.inferenceservice_deployment.parameters.generated.status.state\n",
+    "        inference = data.data.inferenceservice_deployment.parameters.inference\n",
+    "        inference_url = data.data.inferenceservice_deployment.parameters.generated.details.serving.servingurl\n",
+    "        if status == \"RUNNING\":\n",
+    "            break\n",
+    "        print(\"waiting for deployment to be running\")\n",
+    "        time.sleep(api.wait_interval)\n",
+    "    return inference, inference_url"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if USE_REMOTE_DEPLOYMENT:\n",
```

Comment:
Std insurance example doesn't use cloudevents, right? It generates data. Is there a need to distinguish whether it is remote or local deployment?

Reply:
This is not the cloudevent example. It's our  existing example, which can support imported also.